packages/vite/src/resolver.ts

```diff
@@ -52,7 +52,10 @@ export function resolver(): Plugin {
     },
     load(id) {
       if (id.startsWith(virtualPrefix)) {
-        let { src, watches } = virtualContent(id.slice(virtualPrefix.length), resolverLoader.resolver);
+        let { src, watches } = virtualContent(
+          cleanUrlQueryParams(id.slice(virtualPrefix.length)),
+          resolverLoader.resolver
+        );
```

Comment:
The only parameter I have on `load` is the `id`, which corresponds - I guess - to a `result.id` as it contains the query params. I don't see any obvious way to avoid cleaning again the query params before passing to `virtualContent`.